libtrellis/tools/ecppll.cpp

```diff
@@ -50,18 +47,22 @@ struct secondary_params{
   int div;
   int cphase;
   int fphase;
+  const char *name;
```

Comment:
To maintain consistency with the rest of Trellis (and because c_str() can be dangerous), I would prefer C++ strings instead